support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -439,8 +494,66 @@ export function Checkout() {
 										hideDetailsHeading={true}
 										overrideHeadingCopy="1. Your details"
 									/>
+
 									<CheckoutDivider spacing="loose" />
 
+									{product.showAddressFields && (
+										<fieldset>
+											<h2 css={legend}>Where should we deliver to?</h2>
+											<AddressFields
+												scope={'delivery'}
+												lineOne={deliveryLineOne}
+												lineTwo={deliveryLineTwo}
+												city={deliveryCity}
+												country={countryId}
+												state={deliveryState}
+												postCode={deliveryPostcode}
+												countries={product.addressCountries}
+												errors={[]}
+												postcodeState={{
+													results: deliveryPostcodeStateResults,
+													isLoading: deliveryPostcodeStateLoading,
+													postcode: deliveryPostcode,
+													error: '',
+												}}
+												setLineOne={(lineOne) => {
+													setDeliveryLineOne(lineOne);
+												}}
+												setLineTwo={(lineTwo) => {
+													setDeliveryLineTwo(lineTwo);
+												}}
+												setTownCity={(city) => {
+													setDeliveryCity(city);
+												}}
+												setState={(state) => {
+													setDeliveryState(state);
+												}}
+												setPostcode={(postcode) => {
+													setDeliveryPostcode(postcode);
+												}}
+												setCountry={() => {
+													// no-op
+												}}
+												setPostcodeForFinder={() => {
+													// no-op
+												}}
+												setPostcodeErrorForFinder={() => {
+													// no-op
+												}}
+												onFindAddress={(postcode) => {
```

Comment:
Could'nt get this to 'Find Address'? Does it give you more than empty results?

Reply:
This doesn't locally or in `CODE`.

I stubbed it out for testing, but we should look at getting this running in future.